include/ginkgo/core/base/executor.hpp

```diff
@@ -226,7 +226,7 @@ class ExecutorBase;
  * One might feel that this code is too complicated for such a simple task.
  * Luckily, there is an overload of the Executor::run() method, which is
  * designed to facilitate writing simple operations like this one. The method
- * takes three closures as input: one which is run for OMP, one for CUDA
+ * takes four closures as input: one which is run for OMP, one for CUDA
```

Comment:
the changes in this file should be updated in the previous pr, right?

Reply:
Looks like I only updated the LambdaOperation documentation, but not the docs for the actual `run` function.